scripts/invokeJestWithBucklescriptPaths.js

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env node
+// changes path `.ml` or `.mli` to `.bs.js`
+// ex: foo.mli -> foo.bs.js
+
+// 1/ change ext (check)
+// 2/ run `jest`
+
+const path = require('path')
+const {spawnSync} = require('child_process')
+
+// change ext
+
+const pathArr = []
+
+process.argv.forEach(val => {
+   pathArr.push(val)
+})
+
+const newPathArr = pathArr.slice(2)
+
+function pathChanger(arr) {
+   return arr
+      .map(thePath => path.parse(thePath))
+      .filter(pathObj => pathObj['ext'] === '.ml' || pathObj['ext'] === '.mli')
```

Comment:
Try and be consistent about `foo['bar']` vs. `foo.bar`:
```suggestion
      .filter(pathObj => pathObj.ext === '.ml' || pathObj.ext === '.mli')
```